settings.py

```diff
@@ -102,6 +102,12 @@ def show_toolbar_callback(request):
     "SHOW_TOOLBAR_CALLBACK": "settings.show_toolbar_callback",
 }
 
+AES_KEYS = {
```

Comment:
`settings.py`, although the name is misleading, has a comment at the top saying that it's only for development settings. The `lib/settings_base.py` is the default for all configurations. By default there are no encryption keys so it will fail hard. I need to fix up this patch to configure some keys on each production setting file and also file an ops bug to generate the keys once this lands.